src/executables/util_sa.c

```diff
@@ -2860,10 +2860,20 @@ synccoll_check (const char *db_name, int *db_obs_coll_cnt, int *new_sys_coll_cnt
       if (check_tables)
 	{
 	  /* get table names having collation; do not include partition sub-classes */
+
+	  // *INDENT-OFF*
 	  sprintf (query,
-		   "SELECT C.class_name, C.class_type " "FROM _db_class C " "WHERE C.collation_id = %d "
-		   "AND NOT (C.class_name IN " "(SELECT P.class_of.class_name " "FROM _db_partition P WHERE "
-		   "P.class_of.class_name " " = C.class_name AND P.pname IS NOT NULL))", db_coll->coll_id);
+		"SELECT "
+		  "[c].[unique_name] AS [class_name], "
+		  "[c].[class_type] AS [class_type] "
+		"FROM "
+		  "[_db_class] AS [c] "
+		  "LEFT OUTER JOIN [_db_partition] AS [p] ON [p].[class_of] = [c] "
```

Comment:
```suggestion
		  "LEFT OUTER JOIN [_db_partition] AS [p] ON [p].[class_of] = [c] AND [p].[pname] IS NOT NULL"
```

Reply:
This change will change the result.

Reply:
Thank you.
But I thought of a way to remove the join for [_db_partition].

Reply:
Thank you. I will change it.